tests/install_upgrade_operators/conftest.py

```
@@ -151,6 +155,11 @@ def cdi_resource_scope_function(admin_client):
     return get_hyperconverged_cdi(admin_client=admin_client)
 
 
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)